mods/cnc/rules/defaults.yaml

```diff
@@ -593,43 +580,49 @@
 	EditorTilesetFilter:
 		RequireTilesets: DESERT
 
-^Husk:
+^CommonHuskDefaults:
 	Health:
 		HP: 140
 	Armor:
 		Type: Light
+	HiddenUnderFog:
+	AutoTargetIgnore:
+	BodyOrientation:
+
+^Husk:
+	Inherits: ^CommonHuskDefaults
 	Husk:
 		AllowedTerrain: Clear, Rough, Road, Tiberium, BlueTiberium, Beach
-	HiddenUnderFog:
 	Burns:
 		Interval: 2
 	TargetableUnit:
 		RequiresForceFire: yes
 		TargetTypes: Ground
-	AutoTargetIgnore:
 	Capturable:
 		Type: husk
 		AllowAllies: yes
 		CaptureThreshold: 1.0
 	TransformOnCapture:
 		ForceHealthPercentage: 25
-	BodyOrientation:
 	Tooltip:
 		GenericName: Destroyed Vehicle
 	DisabledOverlay:
 	ScriptTriggers:
+	WithFacingSpriteBody:
+	AutoSelectionSize:
+	RenderSprites:
```

Comment:
^CommonHuskDefaults is shared between ^Husk and ^HelicopterHusk so why aren't these three lines shared as well?